src/main/java/io/minio/MinioClient.java

```diff
@@ -382,26 +382,6 @@ public MinioClient(String endpoint, int port, String accessKey, String secretKey
       throw new InvalidEndpointException("(null)", "null endpoint");
     }
 
-    // for valid URL endpoint, port and secure are ignored
-    HttpUrl url = HttpUrl.parse(endpoint);
```

Comment:
Ok, there you should have a version which compiles and tests successfully. In my installation two tests are failing: 

- testSigningKey
- testGetObjectHeaders

But they do even if i try to run the untouched project, i assume that has something to do with my locale, as the code has problems to parse the last modified date correctly.